### PR TITLE
Adding contacnt tracing screen

### PR DESCRIPTION
**Pull request by adnanjt**
This PR has https://github.com/intellisysdcorp/covid-safe-paths/pull/22 Inside also because was started there but we are testing navigation and location screen here

### CODE REVIEW

**Review by RutsSantos — APPROVED** [review]
LGTM. **_Solve conflicts_**

**Review by inteljudajocu — COMMENTED** [review]
Missing socialDistancing.jpg and other assets

File: app/views/DR/ReportScreen/Results.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ export default function Results({ navigation }) {
                   <ResultsContent
                     title='Deberías practicar distancia social'
                     subtitle='En este momento, usted no presenta síntomas de COVID-19, por lo que le recomendamos que continúes con las medidas preventivas de “lavado de manos con agua y jabón de manera frecuente y aplicar el distanciamiento social”'
-                    image={require('../../assets/images/socialDistancing.jpg')}
+                    image={require('../../../assets/images/socialDistancing.jpg')}
```
Comment (inteljudajocu) [inline app/views/DR/ReportScreen/Results.js:134]: This asset is missing

**Review by inteljudajocu — APPROVED** [review]
LGTM, solve conflicts

**Pull request merged**